spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Span.java

```diff
@@ -142,14 +154,14 @@ public synchronized long getAccumulatedMillis() {
 	}
 
 	/**
-	 * Has the span been started and not yet stopped?
+	 * Has the Span been started and not yet stopped?
```

Comment:
this capitalization thing makes it hard to tell what has actually changed. Can you please revert so I can see more easily what's new without reading all javadocs?